plugins/techdocs-backend/api-report.md

```diff
@@ -9,12 +9,13 @@ import express from 'express';
 import { GeneratorBuilder } from '@backstage/techdocs-common';
 import { Knex } from 'knex';
 import { Logger } from 'winston';
+import { PluginCacheManager } from '@backstage/backend-common';
 import { PluginEndpointDiscovery } from '@backstage/backend-common';
 import { PreparerBuilder } from '@backstage/techdocs-common';
 import { PublisherBase } from '@backstage/techdocs-common';
 
 // @public (undocumented)
-export function createRouter({ preparers, generators, publisher, config, logger, discovery, }: RouterOptions): Promise<express.Router>;
+export function createRouter({ preparers, generators, publisher, config, logger, discovery, cache: cacheManager, }: RouterOptions): Promise<express.Router>;
```

Comment:
I'm not 100% sure where `cacheManager` is defined, as we are importing `PluginCacheManager` above. Additionally, are we intending to provide a default value? This might be overridden by `undefined` in previous function signatures when we are always passing `cache` as value, even when it's optional

Reply:
Think this was simplified/clarified as a result of our OutOfTheBox/Recommended typing work a couple weeks back.